compliance.md

```diff
@@ -0,0 +1,11 @@
+# Open Source Software Licence Compliance Note
+
+The BIS has independently authored the software program: SDMX Dashboard Generator. It relies on multiple third-party software modules listed in `requirements.txt`.
+
+SDMX Dashboard Generator is licensed under the Apache License Version 2.0 and is therefore provided with no warranty. To comply with the terms of the licences covering the third-party components, SDMX Dashboard Generator must be installed with the considerations below, any other installation method may not be compliant with the relevant third-party licences.
+
+# Installation considerations
+
+For a licence compliant installation, SDMX Dashboard Generator must be installed using the package installer for Python (pip) using the --no-binary flag. An example installation command is:
+
+`pip install -r requirements.txt--no-binary :all:`
```

Comment:
Please add the following section at the end of the file. Only point (3) of the "Further information" is not needed. The other two points are still required.

```
## Further information

1. Please note that usage of the --no-binary flag will increase the complexity of the installation (such as requiring building modules for some components from source). Please refer to third-party documentation for additional guidance; and

2. Please be aware that compliance materials may be placed into temporary directories by pip;
```